### DISCUSSION TITLE
wrong result for  compare expression of int columns

### DISCUSSION THREAD

**Discussion by ruoyu0088** [perspective-dev/perspective#2514] [Q&A]
1. open the example : https://perspective.finos.org/block/?example=editable
 2. add a new column with title of mask and expression of "Row ID" > 6 
 3. add the mask column to the columns list
 
the values in mask column is not correct:

![image](https://github.com/finos/perspective/assets/826871/ad1aa12c-39b1-49ef-8c69-6b9083e70aa3)

4. change the expression to "Row ID" * 1.00001 > 6 and it works:

![image](https://github.com/finos/perspective/assets/826871/4d20b832-20aa-4fda-9913-dabe01f08343)

**Comment by timkpaine** [comment]
@ruoyu0088 you probably want `float("Row ID") > 6` or `"Row ID" > integer(6)`

**Comment by texodus** [comment]
Duplicate #1614
